under-maintenance/pages/index.tsx

```diff
@@ -36,7 +37,11 @@ export default function RakenteillaSivu() {
         </ContentSpacer>
         <KuvaContainer className="justify-center">
           <Img src="vayla_alla_fi_sv_rgb.png" alt="Väylävirasto logo" sx={{ maxHeight: "117px" }} />
-          <Img src="ely_alla_fi_sv_rgb.png" alt="ELY logo" sx={{ maxHeight: "91px" }} />
+          {isEvkAktivoitu() ? (
+            <Img src="evk_footer_fi_sv.png" alt="Elinvoimakeskus logo" sx={{ maxHeight: "91px" }} />
+          ) : (
+            <Img src="ely_alla_fi_sv_rgb.png" alt="ELY logo" sx={{ maxHeight: "91px" }} />
+          )}
```

Comment:
isEvkAktivoitu -funktion käyttö ei onnistu under-maintenance sivuilla näin. Tää sivu buildataan erikseen ja käytetään staattisena html&css sivuna. Tälle ei päädy tuota ympäristömuuttujaa mitä isEvkAktivoitu käyttää. Sanoisin, et tehdään tässä vaan vertailu näin 
` new Date().getYear() > 125 ? <EVK/> : <ELY/>`